brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumer.java

```diff
@@ -267,11 +264,16 @@ private String determineBestEventReference(S3Event event) {
 
     private UriWrapper constructErrorFileUri(S3Event event,
                                              Exception exception) {
-        var fileUri = UriWrapper.fromUri(extractObjectKey(event));
-        var timestamp = timePath(event);
-        return UriWrapper.fromUri(ERROR_BUCKET_PATH + PATH_SEPERATOR
-                                  + timestamp + PATH_SEPERATOR + exception.getClass().getSimpleName()
-                                  + PATH_SEPERATOR + fileUri.getLastPathElement());
+        return UriWrapper.fromUri(ERROR_BUCKET_PATH)
+                   .addChild(extractInstitutionNameFromRecord())
+                   .addChild(timePath(event))
+                   .addChild(exception.getClass().getSimpleName())
+                   .addChild(UriWrapper.fromUri(extractObjectKey(event)).getLastPathElement());
+    }
+
+    private String extractInstitutionNameFromRecord() {
+        var record = attempt(() -> parseBrageRecordJson(brageRecordFile)).orElseThrow();
+        return record.getCustomer().getName();
```

Comment:
Det er ikke fra rekorden, men fra s3eventen som inneholder institusjonsnavn.

Reply:
I mean we do following:
S3Event -> reading s3 resource uri from event -> fetching s3 resource -> convert s3 resource to record -> reading customer value from record.